scripts/installer/windows/scripts/generate_dependency.py

```diff
@@ -34,28 +49,41 @@ def extract_package_names(packages):
 def get_package_dependencies(package_name_list):
     dependencies = []
     for package_name in package_name_list:
-        result = subprocess.run(['pip', 'show', package_name], stdout=subprocess.PIPE)
+        if (is_tool('conda')):
+            result = subprocess.run('conda activate root | pip show {}'.format(package_name),
+                                    shell=True, stdout=subprocess.PIPE)
+        else:
+            result = subprocess.run(['pip', 'show', package_name], stdout=subprocess.PIPE)
```

Comment:
I think we only can get required package in each promptflow-* now, and can't get packages in [tool.poetry.extras] ?

Reply:
maybe can try get toml extra dependecy like：
```
    # get promptflow-** extra packages
    extra_packages = []
    for package in all_packages:
        if package.startswith('promptflow'):
            data = toml.load(get_git_base_dir() / "src" / package / "pyproject.toml")
            extras = data.get("tool", {}).get("poetry", {}).get("extras", {})
            for _, values in extras.items():
                extra_packages.extend(values)
    extra_packages_names = extract_package_names(extra_packages)
    all_packages.extend(extra_packages_names)
```